web/src/pages/EventPage/EventPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+import { EventService } from "../../services/event";
+import EventList from "../../components/event/EventList";
+import Header from "../../components/common/Header";
+import Footer from "../../components/common/Footer";
+import EventCalendar from "../../components/event/EventCalendar";
+import "./EventPage.scss";
+
+class EventPage extends Component {
```

Comment:
Shouldn't this page be called "EventsPage" or "EventListPage"? Since we'll also have an "EventPage" or "EventDetailPage" right?

Reply:
Right!